app/src/main/java/lt/vilnius/tvarkau/fragments/ProblemsListFragment.java

```diff
@@ -118,7 +118,10 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         unbinder = ButterKnife.bind(this, view);
 
-        swipeContainer.setOnRefreshListener(() -> getData(0));
+        swipeContainer.setOnRefreshListener(() -> {
+            problemList.clear();
```

Comment:
Maybe it's worth creating a method `reloadData()`, because it's a second place where it's needed to do `clear` before `getData(0)`.